### ISSUE TITLE
NVDA fails to obtain states for symbol start menu grouping

### ISSUE THREAD

**Issue by francipvb** [nvaccess/nvda#10668]
### Steps to reproduce:

1. Open the start menu by pressing <kbd>windows</kbd> key.
1. Go to the application list by pressing <kbd>down arrow</kbd>.

* Note that this applies only if you have installed programs whose name starts with a symbol.

### Actual behavior:

NVDA don't speak anything.

The fact is that NVDA cannot obtain states only for this particular object. The rest of the groupings behave as normal. Another consequence is that you cannot use object navigation to go to it.

### Expected behavior:

NVDA should speak the grouping name and role as usual.

#### NVDA installed/portable/running from source:

NVDA installed

#### NVDA version:
alpha-19460,d7229eff

#### Windows version:

Windows 10 pro version 1909 (Build SO 18363.535)

#### Does the issue still occur after restarting your PC?
yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

### Additional log chunk:

Note that I have my locale set to es-ar.

```
IO - inputCore.InputManager.executeGesture (06:22:33.479) - winInputHook (13104):
Input: kb(desktop):downArrow
DEBUGWARNING - Python warning (06:22:33.538) - MainThread (14380):
C:\Program Files (x86)\NVDA\library.zip\keyboardHandler.py:538: DeprecationWarning: Yield() is deprecated
DEBUGWARNING - Python warning (06:22:33.538) - MainThread (14380):
C:\Program Files (x86)\NVDA\library.zip\api.py:287: DeprecationWarning: Yield() is deprecated
DEBUG - editableText.EditableText._hasCaretMoved (06:22:33.678) - MainThread (14380):
Focus event. Elapsed: 40 ms
IO - speech.speak (06:22:33.718) - MainThread (14380):
Speaking ['Todas las aplicaciones', 'lista']
DEBUGWARNING - eventHandler.executeEvent (06:22:33.728) - MainThread (14380):
error executing event: gainFocus on <NVDAObjects.UIA.UIA object at 0x035614B0> with extra args of {}
Traceback (most recent call last):
  File "eventHandler.pyc", line 155, in executeEvent
  File "eventHandler.pyc", line 92, in __init__
  File "eventHandler.pyc", line 100, in next
  File "NVDAObjects\__init__.pyc", line 1102, in event_gainFocus
  File "NVDAObjects\__init__.pyc", line 988, in reportFocus
  File "speech\__init__.pyc", line 485, in speakObject
  File "speech\__init__.pyc", line 318, in speakObjectProperties
  File "baseObject.pyc", line 42, in __get__
  File "baseObject.pyc", line 145, in _getPropertyViaCache
  File "NVDAObjects\UIA\__init__.pyc", line 1215, in _get_states
  File "NVDAObjects\UIA\__init__.pyc", line 800, in _getUIACacheablePropertyValue
  File "comtypesMonkeyPatches.pyc", line 26, in __call__
_ctypes.COMError: (-2147220991, 'Un evento no pudo invocar a ninguno de los subscriptores', (None, None, None, 0, None))
IO - inputCore.InputManager.executeGesture (06:22:35.504) - winInputHook (13104):
Input: kb(desktop):NVDA+f1
INFO - globalCommands.GlobalCommands.script_navigatorObject_devInfo (06:22:35.738) - MainThread (14380):
Developer info for navigator object:
name: '#'
role: ROLE_GROUPING
roleText: None
states: exception: (-2147220991, 'Un evento no pudo invocar a ninguno de los subscriptores', (None, None, None, 0, None))
isFocusable: exception: (-2147220991, 'Un evento no pudo invocar a ninguno de los subscriptores', (None, None, None, 0, None))
hasFocus: False
Python object: <NVDAObjects.UIA.UIA object at 0x035614B0>
Python class mro: (<class 'NVDAObjects.UIA.UIA'>, <class 'NVDAObjects.window.Window'>, <class 'NVDAObjects.NVDAObject'>, <class 'documentBase.TextContainerObject'>, <class 'baseObject.ScriptableObject'>, <class 'baseObject.AutoPropertyObject'>, <class 'object'>)
description: ''
location: RectLTWH(left=75, top=239, width=308, height=44)
value: None
appModule: <'appModuleHandler' (appName 'startmenuexperiencehost', process ID 11324) at address 4e46b30>
appModule.productName: 'Microsoft.Windows.StartMenuExperienceHost'
appModule.productVersion: '10.0.18362.449'
TextInfo: <class 'NVDAObjects.NVDAObjectTextInfo'>
windowHandle: 66088
windowClassName: 'Windows.UI.Core.CoreWindow'
windowControlID: 0
windowStyle: -1811939328
extendedWindowStyle: 2097160
windowThreadID: 11432
windowText: 'Inicio'
displayText: ''
UIAElement: <POINTER(IUIAutomationElement) ptr=0x508c5c0 at 355cee0>
UIA automationID: 
UIA frameworkID: XAML
UIA runtimeID: (42, 66088, 4, 1)
UIA providerDescription: [pid:11324,providerId:0x0 Main(parent link):Unidentified Provider (unmanaged:Windows.UI.Xaml.dll)]
UIA className: ListViewItem
UIA patterns available: LegacyIAccessiblePattern, InvokePattern, ScrollItemPattern, VirtualizedItemPattern
```

**Comment by francipvb** [comment]
CC @leonardder @josephsl @michaelDCurran 

Any thougts?

**Comment by francipvb** [comment]
CC @adriani90

**Comment by CyrilleB79** [comment]
Cannot reproduce here on Win 10 1809 French UI. I can go to the "#" group and when I press NVDA+F1 to get dev info, the state info and depending are displayed correctly (no exception).

**Comment by feerrenrut** [comment]
I can't reproduce this with Win 10 1903 and NVDA 2019.3Beta3

**Comment by Qchristensen** [comment]
Just to double check, @CyrilleB79 are you using 1809 or 1909?

I am using 1903 and I also can't reproduce this.  I wonder if it is something in 1909 (or 1909 in Spanish?)

**Comment by CyrilleB79** [comment]
Yes I confirm that I am using win 10 1809 on my work computer and that the issue described in this PR does not appear.

**Comment by feerrenrut** [comment]
While this seems like an important issue to fix, we have decided not to hold back the release of 2019.3.

**Comment by michaelDCurran** [comment]
This is a bug in the Start Screen's UI Automation implementation. Elements that are off screen (such as when using object navigation or scrolling fast such that focus changes are delayed) seem to have broken UIA element properties. This is pretty harmless and there is nothing we can do about it in NVDA. I don't think its just states that is broken. The entire element becomes unfunctional.

**Comment by Adriani90** [comment]
Should this be closed then as can't fix for now?

**Comment by CyrilleB79** [comment]
@francipvb, is this issue still present with NVDA 2023.3 and recent version of Windows?

**Comment by Adriani90** [comment]
There is no update upon request from the initial author since many years, I am closing this as abandoned and as can't fix, as explained in https://github.com/nvaccess/nvda/issues/10668#issuecomment-600355435

If you are still having this issue, please comment and we can reopen.
